test/Spec.hs

```diff
@@ -78,6 +79,7 @@ legacySpecs = parallel $ do
   describe "Data.Abstract.Path" Data.Abstract.Path.Spec.spec
   describe "Data.Abstract.Name" Data.Abstract.Name.Spec.spec
   describe "Data.Functor.Classes.Generic" Data.Functor.Classes.Generic.Spec.spec
+  describe "Data.Language" Data.Language.Spec.spec
```

Comment:
Would you mind rewriting this as a `tasty` test? We’re trying to move away from `hspec` (as per the comment at the top of `legacySpecs`), since `tasty` plays nicer with the various frameworks we use. Again, happy to 🍐 — the conversion is not hard, and usually more concise than `hspec`

Reply:
Not a problem at all, switched over in 5763cfdfb6173e248e18337ed26f6539b9321853